common/version.go

```diff
@@ -0,0 +1,21 @@
+package common
+
+// version information populated during build time
+var (
+	version   = "v0.2.0" // value from VERSION file
```

Comment:
Can this be `version = "" // value from VERSION file`, or is there an advantage to having `v0.2.0` hardcoded in this file?

Reply:
I thought we needed a default value here, but looks we can always get it from the VERSION file. Will change it to an empty string.